source/docs/solar/nsrdb/spectral_tmy_india_data_download.html.md.erb

```diff
@@ -1,6 +1,6 @@
 ---
 title: Spectral TMY India
-summary: This service has been replaced by a V2 version of the API. See <a href="/docs/solar/nsrdb/spectral-tmy-india-download">Spectral TMY India v2</a>.
+summary: This service has been replaced by a V2 version of the API. See <a href="/docs/solar/nsrdb/spectral-india-tmy-download">Spectral TMY India v2</a>.
 url: /api/nsrdb_api/solar/spectral_tmy_india_download
```

Comment:
@PjEdwards should this path be changed too?

Reply:
No, this is a previous, now deprecated version. I left that path intact so that anyone who used to use the old one could definitively tie it to the new version.